docs/contribute.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to Primer
+
+There are several ways you can contribute to and participate in Primer:
+
+- Participate in discussions
+- Submit an issue
+- Improve the documentation
+- Close issues
+- Propose a new component or UI pattern
+- Questions and support
+
+Please note that all contributions to Primer need to follow our code of conduct.
+
+## Participate in discussions
+
+Discussions about improvements and upcoming work happen mostly in issues and in the #design-system Slack channel. These are the repos you should watch if you’re interested in getting involved:
+
+- Design
+- Primer CSS
+- Primer Components (React)
+- Octicons
+- ...
+
+The Design Systems Team holds office hours twice a week and anyone at GitHub is free to join, ask questions and participate in ongoing conversations.
+
+## Submit an issue 
+
+If you spot something that doesn’t look right in our design, code or documentation, use one of the issue templates to submit an issue.
+
+Before submitting an issue, do a quick search to check the same issue hasn’t been submitted yet. If it has, you can add to the discussion with your own examples and ideas.
+
+
+## Improve the documentation
+
+Documentation is a core part of Primer, and, just as design and code, we’re always trying to make it better and more useful. If you notice something missing, a typo, or have an idea of how the docs can be improved, check if the issue already exists, and, if not, you can do one of the following:
+
+- Submit an issue 
+- Submit a pull request with a fix (you can do this directly via the “Edit this page on GitHub” link on the footer of docs pages)
+
+## Close issues
+
+There are issues marked with the label “status: help wanted” in the design-systems repo that we’d love some help with. If you think you can help, add a comment on the issue stating when and (if you know) how you’ll be able to tackle it and we’ll support you throughout. 
```

Comment:
Refers to a private repo. We want that visibility internally, but is it confusing for everyone else? Should we link to this repo and the main public one?